tools/cldr/cldr-to-icu/src/test/java/org/unicode/icu/tool/cldrtoicu/AlternateLocaleDataTest.java

```diff
@@ -45,7 +45,7 @@ public void testLocaleData() {
 
         FakeDataSupplier src = new FakeDataSupplier()
             .addLocaleData("xx", target, source, other)
-            .addInheritedData("xx", inherited);
```

Comment:
The tests are being changed because I removed the very poor API that let you specify "inherited" data directly rather than adding data to a different locale and having inheritance happen. The original API was prone to all sorts of accidental misuse since you could create very inconsistent data easily.

The "root" locale is hard-coded to be the parent of every parent-less locale, but later on you'll see places where other parent locales are set explicitly.